tests/e2e-utils/README.md

```diff
@@ -0,0 +1,35 @@
+# WooCommerce End to End Test Utilities 
+
+This package contains utilities needed to write e2e tests (specific to WooCommerce).
```

Comment:
Possibly better to say `utilities to simplify writing e2e tests specific to WooCommmerce.` as they aren't required to write tests.